From c38f00576e0d8a4912297d6eebc78ca125a098fd Mon Sep 17 00:00:00 2001
From: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
Date: Fri, 12 Aug 2022 10:44:59 +0800
Subject: [PATCH 48/51] DEPS: update commit hash for ffmpeg

Regenerate the gni files for ffmpeg and
fix ffmpeg linking issue at build time.

Signed-off-by: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
---
 DEPS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/DEPS b/DEPS
index 39762f3a2a48..1568b3135dd5 100644
--- a/DEPS
+++ b/DEPS
@@ -1187,7 +1187,7 @@ deps = {
     Var('chromium_git') + '/external/github.com/google/farmhash.git' + '@' + '816a4ae622e964763ca0862d9dbd19324a1eaf45',
 
   'src/third_party/ffmpeg':
-    Var('reb_git') + '/chromium-ffmpeg.git' + '@' + '63c83e8f91ece09847337856ad7b14a1c25d3c77',
+    Var('reb_git') + '/chromium-ffmpeg.git' + '@' + '54226c9fb8362023b625738519572be1faa079b2',
 
   'src/third_party/flac':
     Var('chromium_git') + '/chromium/deps/flac.git' + '@' + 'af862024c8c8fa0ae07ced05e89013d881b00596',
-- 
2.25.1

